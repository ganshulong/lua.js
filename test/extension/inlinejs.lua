-- Inline Javascript

local log =
    __javascript "console.log"

log("Lua string")

log(__js "1+2")

log(__js("3+" .."4"))
