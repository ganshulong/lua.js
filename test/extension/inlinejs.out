Lua string
3
7
